src/jaegertracing/Config.h

```diff
@@ -17,6 +17,8 @@
 #ifndef JAEGERTRACING_CONFIG_H
 #define JAEGERTRACING_CONFIG_H
 
+#include "jaegertracing/Compilers.h"
+
```

Comment:
Please remove newline here to keep with other Jaeger headers.

Reply:
done